tests/fixtures/2_kakarot.py

```diff
@@ -22,13 +22,15 @@ async def contract_account_class(starknet: Starknet) -> DeclaredClass:
 
 
 @pytest_asyncio.fixture(scope="session")
-async def contract_account(starknet: Starknet):
+async def contract_account(starknet: Starknet, kakarot: StarknetContract):
```

Comment:
this fixture is actually only used in the corresponding `tests/unit/src/kakarot/accounts/contract/test_contract_account.py` file, could you move it there?

Reply:
> 
Sure, I can add these tests.
But from what you said before:
https://github.com/sayajin-labs/kakarot/pull/508#pullrequestreview-1301545454

It sounded to me like the only test we wanted for now is one that checks whether `infinite` is actually `infinite` and doesn't reduce the allowance.